docs/databases/redis/redis-on-ubuntu-14-04.md

```diff
@@ -0,0 +1,173 @@
+---
+author:
+  name: Sergey Pariev
+  email: spariev@gmail.com
+description: 'Deploy applications that depend on the high performance key-value store Redis.'
+keywords: 'redis ubuntu 14.04,redis trusty tahr,nosql,database,key-value store'
+license: '[CC BY-ND 3.0](http://creativecommons.org/licenses/by-nd/3.0/us/)'
+modified: Monday, January 11th, 2016
+modified_by:
+  name: Sergey Pariev
+published: 'Monday, January 11th, 2016'
+title: 'Redis on Ubuntu 14.04 LTS (Trusty Tahr)'
+contributor:
+  name: Sergey Pariev
+  link: https://twitter.com/spariev
+external_resources:
+ - '[Redis Project Home Page](http://redis.io/)'
+ - '[Redis Configuration](http://redis.io/topics/config)'
+ - '[Redis Persistence](http://redis.io/topics/persistence)'
+ - '[Redis Security](http://redis.io/security)'
+---
+
+Redis is an open source in-memory data structure store, with optional disk writes for persistence, which can be used as key-value database, cache and message broker. Redis features built-in transactions, replication, and support for a variety of data structures such as strings, hashes, lists, sets and others. Redis can be made highly available with Redis Sentinel and supports automatic partitioning with Redis Cluster. This document provides both instructions for deploying the Redis server and an overview of best practices for maintaining Redis instances.
+
+## Before You Begin
+
+1.  Familiarize yourself with our [Getting Started](/docs/getting-started) guide and complete the steps for setting your Linode's hostname and timezone.
+
+2.  This guide will use `sudo` wherever possible. Complete the sections of our [Securing Your Server](/docs/security/securing-your-server) to create a standard user account, harden SSH access and remove unnecessary network services.
+
+3.  Update your system.
+
+		sudo apt-get update && sudo apt-get upgrade
+
+
+{: .note}
+>
+>This guide is written for a non-root user. Commands that require elevated privileges are prefixed with `sudo`. If you’re not familiar with the `sudo` command, you can check our [Users and Groups](/docs/tools-reference/linux-users-and-groups) guide.
+
+
+## Redis Installation
+
+Redis package in Ubuntu 14.04 repository is outdated and misses several security patches, so in this case it is better to use third-party PPA.
+
+1.  Add Redis PPA repository to install the latest version:
+
+		sudo add-apt-repository ppa:chris-lea/redis-server
```

Comment:
I did some searching online, and found several references for the PPA `ppa:rwky/redis`. Can you explain the difference between this and the PPA you use in the guide?